src/core/xref.js

```diff
@@ -19,12 +19,12 @@ export async function run(conf, elems) {
   const { xref } = conf;
   const xrefMap = createXrefMap(elems);
   const query = createXrefQuery(xrefMap);
-  const apiURL = new URL(xref.url, location.href) || API_URL;
+  const apiURL = (xref.url && new URL(xref.url, location.href)) || API_URL;
```

Comment:
Here you want:

```JS
const apiURL = xref.url ? new URL(xref.url, location.href).href : API_URL;
```

Reply:
`.href`? need some consistency here
so, which one should change? API_URL or new URL here?